Zend/zend_vm_def.h

```diff
@@ -3042,13 +3042,24 @@ ZEND_VM_HOT_HANDLER(127, ZEND_FE_FREE, TMPVAR, ANY)
 	zval *var;
 	USE_OPLINE
 
-	SAVE_OPLINE();
 	var = EX_VAR(opline->op1.var);
-	if (Z_TYPE_P(var) != IS_ARRAY && Z_FE_ITER_P(var) != (uint32_t)-1) {
-		zend_hash_iterator_del(Z_FE_ITER_P(var));
+	if (Z_TYPE_P(var) != IS_ARRAY) {
```

Comment:
Shouldn't this be UNEXPECTED(...) ? (Maybe now is a good time to do so)

Reply:
Probably - I wasn't sure if there were other uses of the FE_FREE opcode that I hadn't considered.